scripts/kube-init.sh

```diff
@@ -30,6 +30,9 @@ trap "clean_exit" EXIT
 # Switch off SE-Linux
 setenforce 0
 
+# Request at least 2 CPUs from Travis CI per job
+export TRAVIS_WORKER_DOCKER_CPUS=2
```

Comment:
I did some search and it seems the default value for `TRAVIS_WORKER_DOCKER_CPUS` is 2 already. I'm not sure if this is the right fix

https://github.com/travis-ci/worker/blob/b6839eb59412dd46ec2feccc85660c2afd990772/.example.env#L49-L51
https://github.com/travis-ci/worker/blob/b6839eb59412dd46ec2feccc85660c2afd990772/backend/docker.go#L60

For the original issue https://github.com/kubernetes-client/python/issues/890, have you observed other builds in this repo hitting the same error?